pkg/liqo-controller-manager/resource-request-controller/offerUpdater.go

```diff
@@ -134,21 +135,24 @@ func (u *OfferUpdater) CreateOrUpdateOffer(cluster discoveryv1alpha1.ClusterIden
 	})
 
 	if err != nil {
-		klog.Error(err)
+		klog.Errorf("Error during creating or updating offer, requeuing to retry. %s.", err)
```

Comment:
It appears that the callers of this function already print the errors, hence this could be dropped.